Solutions/Salesforce Service Cloud/Analytic Rules/Salesforce-PasswordSpray.yaml

```diff
@@ -0,0 +1,33 @@
+id: 64d16e62-1a17-4a35-9ea7-2b9fe6f07118
+name: Brute force attack against user credentials
+description: |
+  'This query searches for failed attempts to log in from more than 15 various users within a 5 minute timeframe from the same source. This is a potential indication of a password spray attack.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: SalesforceServiceCloud
+    dataTypes:
+      - SalesforceServiceCloud
+queryFrequency: 20m
+queryPeriod: 20m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+relevantTechniques:
+  - T1110
+query: |
+  let FailureThreshold = 15;  
+  SalesforceServiceCloud
+  | where EventType == 'Login'// and  LoginStatus != 'LOGIN_NO_ERROR'
+  | where LoginStatus  in ('LOGIN_ERROR_INVALID_PASSWORD', 'LOGIN_ERROR_SSO_PWD_INVALID')
+  | summarize UserCount=dcount(UserId), Users = make_set(UserId,100) 
+    by ClientIp, bin(TimeGenerated, 5m)
+  | where UserCount > FailureThreshold
+  | extend timestamp = TimeGenerated, IPCustomEntity = ClientIp
+entityMappings:
+  - entityType: IP
+    fieldMappings:
+      - identifier: Address
+        columnName: IPCustomEntity
+version: 1.1.1
```

Comment:
Version should be 1.0.0, correct?

Reply:
Fixed

Reply:
@rushriva  - I don't see the version changed

Reply:
I missed this file, resolved now.